server/api/middlewares/collectionIndexing.js

```diff
@@ -0,0 +1,67 @@
+// @flow
+import fractionalIndex from "fractional-index";
+import { type Context } from "koa";
+import { Collection } from "../../models";
+export default function collectionIndexing() {
+  return async function collectionIndexingMiddleware(
+    ctx: Context,
```

Comment:
Needs to be `ContextWithState` imported from the internal types file to fix your CI error